### ISSUE TITLE
Maintainer & group member

### ISSUE THREAD

**Issue by elmatou** [PiPiper/bcm2835#2]
Maybe I could be a group member in order to speed up a bit the dev.
I do have localy some drivers to create.

see you

**Comment by zsyed91** [comment]
Sorry I should be good to go now, keep those PR's coming :smile: 

What other drivers were you thinking of creating? We can create those repo's and I can try and help you once you push some of your code.

**Comment by elmatou** [comment]
I begun libbcm2835, gave a shot at pi-blaster.... have to work on sysfs.
but I contacted Mike from libbcm2835, and he jst updated the lib with /dev/gpiomem register access. allowing non-root access for his lib ! I'm currently testing it (with no luck for now)

**Comment by zsyed91** [comment]
That's awesome to hear. Let's hope it works, then we no longer have to have
root!

**Comment by zsyed91** [comment]
Ok, I will create that repo.

**Comment by zsyed91** [comment]
https://github.com/PiPiper/sysfs

**Comment by zsyed91** [comment]
Any luck with the non-root driver?

**Comment by elmatou** [comment]
yes partialy. It works after changing the group of /dev/gpiomem. There is some issue in the lenny upgrade. I should check it with a stock image of lenny. That's something we can read here and there.
it definitely will work
